.packit.yaml

```diff
@@ -0,0 +1,33 @@
+# See the documentation for more information:
+# https://packit.dev/docs/configuration/
+actions:
+    changelog-entry:
+        - bash -c 'echo "- New upstream release"'
+    post-upstream-clone:
+        - wget https://src.fedoraproject.org/rpms/rust-zincati/raw/rawhide/f/rust-zincati.spec
+
+specfile_path: rust-zincati.spec
+downstream_package_name: rust-zincati
+upstream_package_name: zincati
+upstream_tag_template: v{version}
+# add or remove files that should be synced
+files_to_sync:
+    - rust-zincati.spec
+    - .packit.yaml
+
+jobs:
+
+- job: propose_downstream
+  trigger: release
+  dist_git_branches:
+    - fedora-all
```

Comment:
Minor: I think this'll make it open against e.g. f38 too, which is totally unnecessary. Is there a way to tell Packit "development releases + only the newest stable release"?

Reply:
Thanks for the review, refer to https://packit.dev/docs/configuration#aliases, `fedora-development` includes `fedora-40` and `fedora-rawhide`, `fedora-latest-stable` includes `fedora-39`, should exclude f38.

Also exclude `f38` for jobs `koji_build` and `bodhi_update`.